src/audio/dai.c

```diff
@@ -97,7 +97,7 @@ static void dai_dma_cb(void *arg, enum notify_id type, void *data)
 	/* stop dma copy for pause/stop/xrun */
 	if (dev->state != COMP_STATE_ACTIVE || dd->xrun) {
 		/* stop the DAI */
-		dai_trigger(dd->dai, COMP_TRIGGER_STOP, dev->direction);
+		dai_trigger_op(dd->dai, COMP_TRIGGER_STOP, dev->direction);
```

Comment:
why and what the `op` postfix means? Would it sound better if they are postfixed instead with `sof`?
`dai_trigger` -> `sof_dai_trigger`

nitpick

Reply:
...and you already use `dai_*_xtos()`, so why not use "xtos" everywhere? And I agree with @ujfalusi - a prefix seems both more common and more logical - you usually go from the least to the most specific part. I.e. "xtos" is the least specific part, "dai" is the next level, and the actual operation is the most specific part

Reply:
This may be a naming challenges we can't win. 

I wonder if it'd be cleaner to rename the public "dai.h" to have "sof_" prefix, and then just have different implementations for this in src/lib/dai.c.

Reply:
yeah totally fine with that, will try and see